cmd/minishift/cmd/util/util.go

```diff
@@ -112,3 +114,28 @@ func IsValidProfileName(profileName string) bool {
 	rr := regexp.MustCompile(`^[a-zA-Z0-9]+[a-zA-Z0-9-]*$`)
 	return rr.MatchString(profileName)
 }
+
+func GetNoProxyConfig(api libmachine.API) (string, string, error) {
```

Comment:
:+1:  Good that now it separated and can be consumed by any command.

Reply:
👍